framework/base/src/types/interaction/result_handlers.rs

```diff
@@ -10,6 +10,7 @@ mod returns_raw_result;
 mod returns_result;
 mod returns_result_as;
 mod returns_result_unmanaged;
+mod token_properties_result;
```

Comment:
Not here, it's not a result handler.

Either in types/io, or in types/interaction/system_proxy, where it is also used